From 3006380ff5e8fd69bc9a7de7364d58f1c86db42e Mon Sep 17 00:00:00 2001
From: Akira Ajisaka <aajisaka@apache.org>
Date: Sat, 15 Oct 2016 22:14:24 +0900
Subject: [PATCH 1932/2748] HDFS-11012. Unnecessary INFO logging on DFSClients
 for InvalidToken. Contributed by Harsh J.

(cherry picked from commit 5ad037df25ab3206509083276b7ef4ef001be48b)
(cherry picked from commit 35832901e87faa8cde687568df3145bea78b4121)
(cherry picked from commit 65e6a1181cb4411741b0805753a0f2ff7c683d9d)

Conflicts:
	hadoop-hdfs-project/hadoop-hdfs-client/src/main/java/org/apache/hadoop/hdfs/DFSInputStream.java

Change-Id: Iaa226b0f159368e46dcc6619b658369090b44ae8
---
 .../org/apache/hadoop/hdfs/DFSInputStream.java     |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/DFSInputStream.java b/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/DFSInputStream.java
index c02eb19..61e05d0 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/DFSInputStream.java
+++ b/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/DFSInputStream.java
@@ -1408,8 +1408,8 @@ private static boolean tokenRefetchNeeded(IOException ex,
      * the estimated expiration date.
      */
     if (ex instanceof InvalidBlockTokenException || ex instanceof InvalidToken) {
-      DFSClient.LOG.info("Access token was invalid when connecting to "
-          + targetAddr + " : " + ex);
+      DFSClient.LOG.debug("Access token was invalid when connecting to "
+          + targetAddr, ex);
       return true;
     }
     return false;
-- 
1.7.9.5

